wallet/src/util/WalletBackendAdapter.ts

```diff
@@ -402,7 +403,13 @@ export const makeWalletBridgeFromFollowers = (
       // If we ever add non 'set' amount purses that aren't in the vbank, it's
       // not currently possible to read their decimalPlaces, so this code
       // will need updating.
-      if (!Array.isArray(purse.balance.value)) {
+
+      const isCopyBag = Object.prototype.hasOwnProperty.call(
```

Comment:
we must have something already in Endo to check this. if not, please file an issue.

Reply:
Indeed 
https://github.com/endojs/endo/blob/66bb24c0226add70fcb6dc01c55418f7582e7c0b/packages/patterns/index.js#L5

Reply:
try https://github.com/endojs/endo/blob/66bb24c0226add70fcb6dc01c55418f7582e7c0b/packages/patterns/index.js#L9C3-L9C12